src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -430,7 +449,11 @@ private BoundExpression CheckValue(BoundExpression expr, BindValueKind valueKind
                     return expr;
 
                 case BoundKind.IndexerAccess:
-                    expr = BindIndexerDefaultArgumentsAndParamsCollection((BoundIndexerAccess)expr, valueKind, diagnostics);
+                    expr = BindIndexerDefaultArgumentsAndParamsCollection((BoundIndexerAccess)expr, valueKind, diagnostics
+#if DEBUG
```

Comment:
Thought we were going to remove these.

Reply:
@jaredpar 

> Thought we were going to remove these.

We still are. That change however doesn't affect the behavior and I wanted to get something out for review sooner.